v5/core/base_service_test.go

```diff
@@ -1496,6 +1496,21 @@ func TestAuthNotConfigured(t *testing.T) {
 	assert.Nil(t, service)
 }
 
+func TestAuthInterfaceNilValue(t *testing.T) {
+	// Verify a non-nil service is cloned correctly.
+	authenticator := &NoAuthAuthenticator{}
+	authenticator = nil
```

Comment:
this actually causes a lint error in the travis build :)
you could just pass nil directly in the struct literal below, or change the above 2 lines to be something like this:
```
        var authenticator *NoAuthAuthenticator = nil
```

Reply:
Sorry, I forgot to run the linter in local! Fixed now.